dominos/src/main/java/me/fontys/semester4/dominos/ImportAll.java

```diff
@@ -0,0 +1,97 @@
+package me.fontys.semester4.dominos;
```

Comment:
Why delete the mainfile from master and create a new one? This will conflict with other people's pull requests. Please keep the original and make changes there to keep the Git history clean and manageable.

Reply:
Yes, that's my OCD kicking in. I could not ignore it for that long. Agreed though. Back to my senses. Fixed. Push inc.